### ISSUE TITLE
rlinks: Implement `stop_link`

### ISSUE THREAD

**Issue by om26er** [crossbario/crossbar#1908]
The `stop_link` API is not implemented. This can be very useful when writing tests for rlinks.

Imagine a static router config with three different router workers, all started but not connected to each other. The ability to start and stop links dynamically could make that we can tests a whole bunch of different scenarios, without too much hassle.

**Comment by om26er** [comment]
Here is the relevant API https://github.com/crossbario/crossbar/blob/b9799d2ac6667a97373db0549d2055fff97d784c/crossbar/worker/rlink.py#L891

**Comment by oberstet** [comment]
Yes, that would be useful to have, agreed. Eg it would also allow to orchestrate more complex integration tests ... some already exist in commented form, eg https://github.com/crossbario/crossbar/blob/b9799d2ac6667a97373db0549d2055fff97d784c/test/test_cli_1.sh#L104
